Makefile

```diff
@@ -7,12 +7,12 @@ PROTOC_GEN_GO_VERSION:=v1.34.2
 PROTOC_GEN_GO_GRPC_VERSION:=v1.5.1
 
 # add any components here after their directory has been created with .proto files
-COMPONENTS:=connector device 
+COMPONENTS:=common connector device 
```

Comment:
don't need common here since its being mounted as a separate volume below.

Reply:
but we still need to built it to get the go code.